burn-train/src/metric/dashboard/base.rs

```diff
@@ -199,8 +213,8 @@ where
     M: Metric + Numeric + 'static,
     T: Adaptor<M::Input>,
 {
-    fn update(&mut self, item: &LearnerItem<T>) -> (MetricEntry, f64) {
-        let update = self.metric.update(&item.item.adapt());
+    fn update(&mut self, item: &LearnerItem<T>, metadata: &MetricMetadata) -> (MetricEntry, f64) {
```

Comment:
for consistency, should use `LearningRate` instead of 64.

Reply:
Same here